### DISCUSSION TITLE
C or C++?

### DISCUSSION THREAD

**Discussion by edm2k** [Schweinepriester/github-profile-achievements#327] [Q&A] [answered]
C or C++?

**Comment by devEyosiyas** [comment, marked as answer]
C tends to be use low-level mission critical stuff such as kernels and embedded systems. C++ is used for more complex applications like browsers, games, or professional productivity tools.

**Comment by M4rcxs** [comment]
C was my first language, so i prefer C XD

**Comment by chenny30** [comment]
Choose Matlab, bro!!!
